src/test/resources/liquibase/harness/change/expectedSql/cloudspanner/mergeColumns.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE full_name_table (id INT64 NOT NULL, first_name STRING(50), last_name STRING(50)) PRIMARY KEY (id)
+INSERT INTO full_name_table (id, first_name) VALUES (1, 'John')
+UPDATE full_name_table SET last_name = 'Doe' WHERE first_name='John'
+INSERT INTO full_name_table (id, first_name) VALUES (2, 'Jane')
+UPDATE full_name_table SET last_name = 'Doe' WHERE first_name='Jane'
```

Comment:
Is there a specific reason that this test executes an INSERT and an UPDATE statement for each row, instead of just inserting all data in one INSERT statement?